File: IV. Fourth Year/net-dotnet-development/tsi-net-languages/InputThird/Class1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace InputThird
{
    public class Class1
    {
        public int GetInput()
        {
            int i = 0;
            bool success = false;

            do
            {
                Console.Write("C#: Please enter an integer> ");
                success = Int32.TryParse(Console.ReadLine(), out i);
            } while (!success);

            return i;
        }
    }
}

File: IV. Fourth Year/net-dotnet-development/tsi-net-languages/InputSecond/Class1.vb
Public Class Class1
    Function GetInput()
        Dim i = 0
        Dim status = False

        Do
            Console.Write("VB: Please enter an integer> ")
            status = Int32.TryParse(Console.ReadLine(), i)
        Loop While Not status

        Return i
    End Function
End Class
